dashboard/app/views/scripts/show.html.haml

```diff
@@ -1,5 +1,5 @@
 = render partial: 'shared/check_admin'
-#landingpage
+#landingpage{class: @script.professional_learning_course ? 'teacher-course' : nil}
```

Comment:
i wonder if it makes sense to pass a parameter through to the view rather than have the view peek at the script?  probably not, but just a thought.